packages/backend/src/accounting/tigerbeetle/utils.ts

```diff
@@ -9,6 +9,9 @@ export function toTigerbeetleId(id: AccountId): bigint {
   if (typeof id === 'bigint') {
     return id
   }
+  if (typeof id === 'string' && !isNaN(Number(id))) {
+    return BigInt(id)
+  }
```

Comment:
Allows numerical, non-UUID strings to be used as an accountId. Basically, this allows `asset.ledger` to be passed in as an id